compiler/AST/build.cpp

```diff
@@ -512,17 +527,27 @@ BlockStmt* buildRequireStmt(CallExpr* args) {
     Expr* useArg = expr->remove();
 
     //
-    // if this is a string argument to 'require', process it
+    // if this is a string literal, process it if we should
     //
     if (const char* str = toImmediateString(useArg)) {
-      processStringInRequireStmt(str);
+      if (processStringInRequireStmt(str)) {
```

Comment:
I agree.  I thought of this after filing the PR, but then figured I'd wait and see what the review said first.  Thanks!